tools/tool-testing/clients/browserstack/clients.js

```diff
@@ -46,6 +46,7 @@ module.exports = [
     selenium: {
       platform: "MAC",
       browserName: "safari",
+      version: "11",
```

Comment:
Read about this change here https://github.com/meteor/meteor/issues/11064#issuecomment-696216504

This is a temporary change.